### DISCUSSION TITLE
Install script does not respect PDM_USE_VENV env variable

### DISCUSSION THREAD

**Discussion by g-kartik** [pdm-project/pdm#3058] [General]
I am using pdm in multi-stage docker file for my web application. I just want to use `pdm` to export the lock file to requirements.txt, so that I can build wheels in the build stage and install them in the run-time stage.  Following the recommended way to install `pdm`, I tried the following:
```
    curl -sSLO https://pdm-project.org/install-pdm.py \
    && curl -sSL https://pdm-project.org/install-pdm.py.sha256 | shasum -a 256 -c - \
    && PDM_USE_VENV=False python3 install-pdm.py --version ${PDM_VERSION}
```
My issues are:
- Why the install script tries to create virtual environment by default? In my opinion it should just install `pdm` just like when install using `pip`
- Even if there are some good reasons, it is not respecting the `PDM_USE_VENV=False` env variable.

**Comment by frostming** [comment]
The installation script always installs PDM in an virtualenv and isn't controlled by `PDM_USE_VENV` to ensure isolation.

If you don't want it to be isolated just install with pip, thanks
